### ISSUE TITLE
Create AbstractTestFixtureAttribute

### ISSUE THREAD

**Issue by GoppeltM** [nunit/nunit#95]
Discovered with NUnit 2.6.3 , NUnit Test Adapter 1.0

Parametrizing TestFixtures may lead to invalid tests.
Let's imagine the following code:

```
Class Parent

    Sub New(s As String)

    End Sub

    <Test>
    Sub doSth()

    End Sub

End Class

<TestFixture("Alpha")>
<TestFixture("Beta")>
Class Child
    Inherits Parent

    Sub New(s As String)
        MyBase.New(s)
    End Sub
End Class
```

After building, this results in the following tests:
Child("Alpha").doSth
Child("Beta").doSth
Parent.doSth

Executing the last test will always result in an error ("No suitable constructor was found"), because apparently a parameterless TestFixture is assumed for the base class,

You can circumvent this phenomenon by declaring the base class as abstract. However, I doubt that this is a proper solution. I encountered this issue while writing test classes that required deep inheritance structures. At one point, I tried to instanciate a base class for composition with specific constructor parameters. But because I was forced to make it abstract this wasn't possible.
In other words: I think enforcing an abstract base class is too limiting. A class may serve as test base without being considered as a test. I can understand that it is convenient that TestFixture may get inherited to each child, but I think it is strange that decorating a child as TestFixture automatically marks the base class as TestFixture as well.

**Comment by rprouse** [comment]
Nobody has picked this up and nobody else has asked for this so closing.
